core/java/iesi-core/src/test/java/io/metadew/iesi/script/execution/instruction/data/date/ListSizeTest.java

```diff
@@ -79,4 +79,23 @@ void generateOutputWithNull() {
         assertThrows(IllegalArgumentException.class, () -> listSize.generateOutput(null));
 
     }
+
+    @Test
+    void generateOutputWithExceptionMSG() {
+        doReturn(new Null()).when(dataTypeHandlerServiceSpy).resolve(null, executionRuntime);
+        listSize = new ListSize(executionRuntime);
+        Exception exception = assertThrows(IllegalArgumentException.class, () -> listSize.generateOutput(null));
+        assertThat(exception.getMessage()).isEqualTo("list cannot be of type class io.metadew.iesi.datatypes._null.Null");
```

Comment:
you can combine these tests using assertJ
https://www.baeldung.com/assertj-exception-assertion